ImperatorToCK3/Source/Mappers/TagTitleMapper/TagTitleMapping.h

```diff
@@ -0,0 +1,26 @@
+#ifndef TAG_TITLE_MAPPING_H
+#define TAG_TITLE_MAPPING_H
+
+#include "Parser.h"
+#include <set>
+
+namespace mappers
+{
+class TagTitleMapping: commonItems::parser
+{
+  public:
+	TagTitleMapping() = default;
+	explicit TagTitleMapping(std::istream& theStream);
+
+	[[nodiscard]] std::optional<std::string> tagRankMatch(const std::string& imptag, const std::string& rank) const;
```

Comment:
imptag -> impTag

If you're aiming to avoid a name collision, you can name the member variable impTag_

Reply:
Renamed impTag to imperatorTag and imptag to impTag.